.env.example

```
@@ -4,8 +4,14 @@ LLM_API_PORT=8001
 CLIENT_PORT=3000
 CLIENT_URL=http://localhost:${CLIENT_PORT}
 OPENAI_API_BASE_URL=http://localhost:${LLM_API_PORT}/v1
+OCR_API_PORT=8005
+OCR_API_BASE_URL=http://localhost:8005/v1
 OPENAI_API_KEY='none'
 LLM_MODEL='ISTA-DASLab/gemma-3-27b-it-GPTQ-4b-128g'
 HMAC_SECRET='your_hmac_secret' # generate with openssl rand 32 | base64
 DOCLING_URL='http://localhost:8004/v1'
 WHISPER_URL='http://localhost:50001/v1'
+OPTIMIZER_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OPTIMIZER_API_KEY key should go before the WHISPER_URL key

(UnorderedKey)